client/src/app/globals.css

```
@@ -349,3 +349,53 @@ template {
 [hidden] {
   display: none;
 }
+
+/* Primitives
+   ========================================================================== */
+:root {
+  --default-font-family: "SUIT-Regular", sans-serif;
+  --default-font-style: normal;
+  --default-line-height: normal;
+  --spacing05: 4px;
+  --spacing1: 8px;
+  --spacing2: 16px;
+  --spacing3: 24px;
+  --spacing4: 32px;
+  --spacing5: 40px;
+  --spacing6: 48px;
+  --spacing7: 56px;
+  --spacing8: 64px;
+  --spacing9: 72px;
+  --spacing10: 80px;
+  --spacing11: 88px;
+  --spacing12: 96px;
+  --spacing13: 104px;
+  --radius-sm: 4px;
+  --radius-md: 8px;
+  --radius-lg: 16px;
+  --radius-xl: 24px;
+  --radius-2xl: 32px;
+  --radius-3xl: 40px;
+  --gray50: #f1f1f1;
```

```suggestion
  --primitives_color_gray-50: #f1f1f1;
```

요것도 primitive이니 prefix 붙여봄. color는 생략해도 되려나?